boards/nrf6310/include/board.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    board_pca10005 PCA10005 (nRF51822 Development Kit)
+ * @ingroup     boards
+ * @brief       Board specific files for the nRF51822 board pca10005
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the nRF51822 evaluation board pca10005
+ *
+ * @author      Christian Kühling <kuehling@zedat.fu-berlin.de>
+ * @author      Timo Ziegler <timo.ziegler@fu-berlin.de>
+ */
+
+#ifndef __BOARD_H
+#define __BOARD_H
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Define the nominal CPU core clock in this board
+ */
+#define F_CPU               (16000000UL)
+
+/**
+ * @name Define the boards stdio
+ * @{
+ */
+#define STDIO               UART_0
+#define STDIO_BAUDRATE      (115200U)
+#define STDIO_RX_BUFSIZE    (64U)
+/** @} */
+
+/**
+ * @name Assign the hardware timer
+ */
+#define HW_TIMER            TIMER_0
+
+/**
+ * @name Macros for controlling the on-board LEDs.
+ * @{
+ */
+#define LED_0_PIN           8
+#define LED_1_PIN           9
+#define LED_2_PIN           10
+#define LED_3_PIN           11
```

Comment:
This can be used by an application. This Board comes with LEDs an Buttons. When i cannot define more than three LEDs here, where is the correct location to define hard wired components like LED's and Buttons? Whats the correct naming schema?